# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xaos/impicit-declarations.gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- XaoS-3.1/src/ui/ui.c.orig	2004-12-07 12:24:38.714132219 +0100
+++ XaoS-3.1/src/ui/ui.c	2004-12-07 12:27:32.562769583 +0100
@@ -168,7 +168,11 @@
    "exact size of one pixel in centimeters"},
   {NULL, 0, NULL, NULL}
 };
-
+
+/*  forward declarations  */
+static void ui_mkimages (int w, int h);
+static void main_loop (void);
+
 static int resizeregistered = 0;
 static void
 ui_updatemenus (uih_context * c, CONST char *name)
